src/analysis/souffle/tests/arcs_fact_tests/fact_test_driver_with_check_facts.cc

```diff
@@ -0,0 +1,91 @@
+//-----------------------------------------------------------------------------
```

Comment:
We have one too many test drivers. :) Why could we not reuse the `fact_test_driver` itself?

Reply:
OK, good point, I'll get rid of the new one and use one test driver at a time.

Reply:
Fixed.